doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -471,6 +471,39 @@ Field              | Type       | Meaning
 `cpu_nano_cores`   | [`bigint`] | Approximate CPU usage, in billionths of a vCPU core.
 `memory_bytes`     | [`bigint`] | Approximate RAM usage, in bytes.
 
+### `mz_storage_source_statistics`
+
+The `mz_storage_source_statistics` table gives the last known storage-specific statistics
+for all processes of all extant storage sources.
+
+At this time, we do not make any guarantees about the exactness or freshness of these numbers.
```

Comment:
```suggestion
Materialize does not make any guarantees about the exactness or freshness of these numbers.
```

Reply:
I think this is the place to mention that these statistics periodically reset to 0 as internal components of the system restart.